model/src/main/kotlin/LicenseFinding.kt

```diff
@@ -39,6 +39,11 @@ data class LicenseFinding(
      */
     val location: TextLocation
 ) : Comparable<LicenseFinding> {
+    /**
+     * Optionally : describe where the license is coming from, using scanner-specific information
+     */
+    var source: String? = null
```

Comment:
This property should be part of the constructor, otherwise it is not respected by functions generated for the data class (`hashCode`, `equals`, `toString`). And please add `@JsonInclude(JsonInclude.Include.NON_NULL)` to not serialize this if it is `null`.

Can you give an example for what strings the FossId scanner would add to this field?

Reply:
done

Reply:
The idea would be to potentially trigger different evaluator rules depending on whether the `LicenseMatchType` is `FILE` or `SNIPPET`. But I agree this could be added at a later point once we have the first concrete need in a rule for it.

Reply:
I removed it and will do a separate PR.

My understanding was that the scan-result.yml was in itself an endpoint, no just an intermediate step to the Reporter or the Evaluator.
Therefore, if an interesting result from my scanner is missing, we could just add it to the data model.

Currently I found the current LicenseFinding to not be extensible enough : there is just the license name and the position. 
1. the position is not relevant with FossID because the scanner does not provide this information.
2. FossId is giving me additional information (the "source") which I cannot put in the finding.
3. the license finding is not inheritable (data class).

In my opinion, there should be base properties common to all scanners, but also extension points to add scanner-specific information.
But this discussion will be the topic of the other PR.

done.